client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts

```diff
@@ -587,7 +587,8 @@ export const meetingSettings: SettingsGroup[] = fillInSettingsDefaults([
                     },
                     {
                         key: `motions_create_enable_additional_submitter_text`,
-                        label: _(`Enable text-based submitter in motion creation`),
+                        label: _(`Activate text-based submitter field in motion creation`),
+                        helpText: _(`Requires permission to manage motions`),
```

Comment:
```suggestion
                        helpText: _(`Requires permission to manage motion metadata`),
```
That you need motion.can_manage is a bug 
It is wanted that you can use this feature if you have motion.can_manage_metadata
Do you already want this corrected here or should this be corrected while fixing this Issue: 
https://github.com/OpenSlides/openslides-client/issues/4457

Reply:
we fix this later if #4457 is merged